MAESTRO/Snakemake/scRNA/rules/sc_rna_map.smk

```diff
@@ -16,11 +16,11 @@ if config["platform"] == "10x-genomics":
         output:
             bam = "Result/STAR/{sample}/{sample}Aligned.sortedByCoord.out.bam",
             bai = "Result/STAR/{sample}/{sample}Aligned.sortedByCoord.out.bam.bai",
-            rawmtx = "Result/STAR/{sample}/{sample}Solo.out/Gene/raw/matrix.mtx",
-            feature = "Result/STAR/{sample}/{sample}Solo.out/Gene/raw/features.tsv",
-            barcode = "Result/STAR/{sample}/{sample}Solo.out/Gene/raw/barcodes.tsv"
+            rawmtx = "Result/STAR/{sample}/{sample}Solo.out/" + config["STARsolo_Features"] + "/raw/matrix.mtx",
```

Comment:
if one specifies` --soloFeatures Gene GeneFull`. how does here work?  it will have two folders:
Gene/raw/matrix.mtx and GeneFull/raw/matrix.mtx?